### PR TITLE
SQL Server and Oracle adapters workaround

### PR DESCRIPTION
**Pull request by grzegorzblaszczyk**
SQL Server and Oracle adapters workaround

### CODE REVIEW

**Comment by jkowens** [comment]
Does an error currently occur if there is an Oracle SQLServer connection configured for the application? I thought there had been code added that prevented that from happening.

File: lib/activerecord-import/base.rb [hunk 1]
```diff
@@ -12,17 +12,22 @@ def self.base_adapter(adapter)
...
-      require File.join(ADAPTER_PATH, "/#{base_adapter(adapter)}_adapter")
-    rescue LoadError
-      # fallback
+    base_adapter = base_adapter(adapter)
+    unless base_adapter.blank?
+      require File.join(ADAPTER_PATH, "/abstract_adapter")
+      begin
+        require File.join(ADAPTER_PATH, "/#{base_adapter(adapter)}_adapter")
+      rescue LoadError
+        # fallback
```
Comment (jkowens) [inline lib/activerecord-import/base.rb]: Doesn't this rescue clause catch any errors if activerecord-import doesn't support an adapter?

Comment (grzegorzblaszczyk) [inline lib/activerecord-import/base.rb]: Yes, you are right - it should. 

This code should also at least log some error - rescueing LoaderError without any sign does not give you any clue.

**Comment by jkowens** [comment]
I still don't understand the purpose of this PR. It doesn't appear to really change any behavior. It silently skips unsupported adapters, which the existing code already does.

**Comment by grzegorzblaszczyk** [comment]
I tried to merge and do a pull request on one of our old forks at https://github.com/goodylabs/activerecord-import/commit/8d2462a629dd4f5a277799186070b51ff5142b2c to have all code used by goodylabs to be potentially in your master branch.

This change was done by my collegue, Adam Migodziński, on 3 Jun 2013, so probably way before you introduced the new way of handling unsupported adapters.

I think it's safe to close this PR and forget about the whole thing. It should be already handled by a new code of yours.

**Comment by jkowens** [comment]
Ohh yeah, I believe this had been fixed since then. Thanks for thinking to give back 👍

**Pull request closed without merging**